CDP4WebServices.API/AutoGenServices/ActionItemService.cs

```diff
@@ -210,6 +210,32 @@ public bool DeleteConcept(NpgsqlTransaction transaction, string partition, Thing
             return this.ActionItemDao.Delete(transaction, partition, thing.Iid);
         }
 
+        /// <summary>
+        /// Delete the supplied <see cref="ActionItem"/> instance.
+        /// A "Raw" Delete means that the delete is performed without calling before-, or after actions, or other side effects.
+        /// This is typically used during the import of existing data to the Database.
+        /// </summary>
+        /// <param name="transaction">
+        /// The current <see cref="NpgsqlTransaction"/> to the database.
+        /// </param>
+        /// <param name="partition">
+        /// The database partition (schema) from where the requested resource will be removed.
+        /// </param>
+        /// <param name="thing">
+        /// The <see cref="ActionItem"/> to delete.
+        /// </param>
+        /// <param name="container">
+        /// The container instance of the <see cref="ActionItem"/> to be removed.
+        /// </param>
+        /// <returns>
+        /// True if the removal was successful.
+        /// </returns>
+        public bool RawDeleteConcept(NpgsqlTransaction transaction, string partition, Thing thing, Thing container = null)
+        {
+
```

Comment:
Unneeded extra newline.

Reply:
True, but I don't want to delay things more at this point in time.